packages/@okta/vuepress-theme-prose/components/ThemeModeToggle.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <label class="toggle-switch switch-theme">
+    <span :class="{'light-mode': true, 'active': !isDarkMode}" @click="toggleDarkMode(false)"></span>
+    <span :class="{'dark-mode': true, 'active': isDarkMode}" @click="toggleDarkMode(true)"></span>
+  </label>
+</template>
+
+<script>
+import { isSetCookie, getCookie, setCookie } from "../util/attribution/cookies";
+
+const themeModeCookieName = 'is_dark_mode';
+
+export default {
+  data() {
+    return {
+      isDarkMode: 
```

Comment:
This has probably been discussed and I'm just unaware, are we planning on using the OS level setting as well?

Reply:
Yes, we have discussed it ad next line checks system settings
window.matchMedia('(prefers-color-scheme: dark)').matches